src/panels/config/voice-assistants/debug/assist-pipeline-run-debug.ts

```diff
@@ -179,6 +190,83 @@ export class AssistPipelineRunDebug extends LitElement {
     );
   }
 
+  private async _runAudioWakeWordPipeline() {
+    const audioRecorder = new AudioRecorder((data) => {
+      if (this._audioBuffer) {
+        this._audioBuffer.push(data);
+      } else {
+        this._sendAudioChunk(data);
+      }
+    });
+
+    this._audioBuffer = [];
+    await audioRecorder.start();
+
+    let run: PipelineRun | undefined;
+
+    let stopRecording: (() => void) | undefined = () => {
+      stopRecording = undefined;
+      audioRecorder.close();
+      // We're currently STTing, so finish audio
+      if (run?.stage === "stt" && run.stt!.done === false) {
+        if (this._audioBuffer) {
+          for (const chunk of this._audioBuffer) {
+            this._sendAudioChunk(chunk);
+          }
+        }
+        // Send empty message to indicate we're done streaming.
+        this._sendAudioChunk(new Int16Array());
+      }
+      this._audioBuffer = undefined;
+    };
+
+    await this._doRunPipeline(
+      (updatedRun) => {
+        run = updatedRun;
+
+        // When we start wake work stage, the WS has a binary handler
+        if (updatedRun.stage === "wake_word" && this._audioBuffer) {
+          // Send the buffer over the WS to the Wake Word / STT engine.
+          for (const buffer of this._audioBuffer) {
+            this._sendAudioChunk(buffer);
+          }
+          this._audioBuffer = undefined;
+        }
+
+        // Stop recording if the server is done with STT stage
+        if (
+          !["ready", "wake_word", "stt"].includes(updatedRun.stage) &&
+          stopRecording
+        ) {
+          stopRecording();
+        }
+
+        // Play audio when we're done.
+        if (updatedRun.stage === "done") {
+          const url = updatedRun.tts!.tts_output!.url;
```

Comment:
We can received this payload : 
```json
{
    "init_options": {
        "start_stage": "wake_word",
        "end_stage": "tts",
        "input": {
            "sample_rate": 48000
        },
        "pipeline": "01gyy99epcd8ngts0a2k167xaq",
        "conversation_id": null
    },
    "stage": "done",
    "run": {
        "pipeline": "01gyy99epcd8ngts0a2k167xaq",
        "language": "fr",
        "runner_data": {
            "stt_binary_handler_id": 1,
            "timeout": 30
        }
    },
    "events": [
        {
            "type": "run-start",
            "data": {
                "pipeline": "01gyy99epcd8ngts0a2k167xaq",
                "language": "fr",
                "runner_data": {
                    "stt_binary_handler_id": 1,
                    "timeout": 30
                }
            },
            "timestamp": "2023-09-13T07:40:24.752343+00:00"
        },
        {
            "type": "wake_word-start",
            "data": {
                "engine": "wake_word.openwakeword",
                "metadata": {
                    "format": "wav",
                    "codec": "pcm",
                    "bit_rate": 16,
                    "sample_rate": 16000,
                    "channel": 1
                }
            },
            "timestamp": "2023-09-13T07:40:24.752479+00:00"
        },
        {
            "type": "error",
            "data": {
                "code": "wake-word-timeout",
                "message": "Wake word was not detected"
            },
            "timestamp": "2023-09-13T07:40:32.936110+00:00"
        },
        {
            "type": "run-end",
            "data": null,
            "timestamp": "2023-09-13T07:40:32.936201+00:00"
        }
    ],
    "wake_word": {
        "engine": "wake_word.openwakeword",
        "metadata": {
            "format": "wav",
            "codec": "pcm",
            "bit_rate": 16,
            "sample_rate": 16000,
            "channel": 1
        },
        "done": false
    },
    "error": {
        "code": "wake-word-timeout",
        "message": "Wake word was not detected"
    }
}
```

I'm not sure if it's a back end issue or not but the stage is `done` and there is an error. Shouldn't the state be `error`?